### ISSUE TITLE
Any updated to MV3 planned?

### ISSUE THREAD

**Issue by cryptoKevinL** [MetaMask/extension-provider#28]
Hi, are there any examples of this package used in Manifest V3 or are they any plans to upgrade this to MV3? I'm struggling to use Metamask in my MV3 application without the provider breaking my alarms, and general things used in the new MV3 background worker.  Thanks!

**Comment by kevinghim** [comment]
@cryptoKevinL Yes, we're currently working on upgrading the manifest version to v3 for Chrome.

**Comment by cryptoKevinL** [comment]
Cool!  thanks so much @kevinghim - any ETA on that?  1 month away, 6 months away?  Really struggling with the use of MM in my Chrome Extension - just declaring the provider kills all my alarms that normally run in the background just fine when the extension is closed.  There must be some error downstream I can't find thats not being bubbled up to the main debug windows

**Comment by kevinghim** [comment]
@cryptoKevinL no specific ETA, but we're planning on migrating to MV3 well before the Jan '23 deadline

**Comment by canokaue** [comment]
Hey @kevinghim just bumping this up once again. Any approx ETAs for V3? Would you guys like some support?

**Comment by croath** [comment]
Should this issue be associated with https://github.com/MetaMask/metamask-extension/milestone/69 ?

**Comment by david-dacruz** [comment]
Hi any updates on the release?

**Comment by samstr** [comment]
The deadline has passed.
